demos/typescript-app/webpack.config.js

```diff
@@ -0,0 +1,21 @@
+const path = require('path');
+
+module.exports = {
+  entry: './src/index.ts',
+  module: {
+    rules: [
+      {
+        test: /\.tsx?$/,
+        use: 'ts-loader',
+        exclude: /node_modules/,
+      },
+    ],
+  },
+  resolve: {
+    extensions: ['.tsx', '.ts', '.js'],
+  },
+  output: {
+    filename: 'bundle.js',
+    path: path.resolve(__dirname, 'dist'),
+  },
```

Comment:
Please refer to the root webpack.config setting.
It seems there are some unnecessary description. (ex. tsx)

I think it's ok with main.js as filename as same as esm-demo-app. How do you think?

Reply:
Yes, I will refer and update it